backend/backend/settings/deps/ledger.py

```diff
@@ -17,7 +17,7 @@
 LEDGER_PEER_TLS_CLIENT_KEY = '/var/hyperledger/tls/client/pair/tls.key'
 LEDGER_PEER_TLS_CLIENT_CERT = '/var/hyperledger/tls/client/pair/tls.crt'
 
-LEDGER_CLIENT_STATE_STORE = '/tmp/hfc-cvs'
+LEDGER_CLIENT_STATE_STORE = os.getenv('LEDGER_CLIENT_STATE_STORE', '/tmp/hfc-cvs')
```

Comment:
`/usr/share` would be more appropriate for this kind of data, see [FHS spec](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html) about it.

Reply:
`/usr/share` cannot be use inside the container because of running the container with a user who does not have the right to write in it.
Moreover you can see my answer to Aurélien below too :)